benchmark/all.py

```diff
@@ -66,11 +68,24 @@ def all_perf_configurations(args: argparse.Namespace) -> List[perf_system.PerfCo
     Set args for all perf configurations.
     """
     args.http1 = True
+    args.http2 = True
```

Comment:
What does both set to true mean in this case? I'm assuming the benchmark is run twice for both HTTP/1 and HTTP/2 by default?